tests/run_tests.py

```diff
@@ -43,6 +44,10 @@
     parser.print_help()
     exit(1)
 
+if option.test_asan and (option.genericjs or option.preexecute or option.all):
+    print("--asan is not supported with --genericjs/--preexecute/--all", file=sys.stderr)
```

Comment:
it would be nice if --all worked with asan.
Can't we make it so that if you specify --all, then the asmjs/wasm tests are run with asan?

Reply:
or explicitly --all --asan maybe